erpnext_documentation/www/docs/user/manual/en/human-resources/employee-other-income.md

```diff
@@ -0,0 +1,28 @@
+<!-- add-breadcrumbs -->
+# Employee Other Income
+
+
+**Employee Other Income is a document to declare other incomes of an employee from different sources.**
```

Comment:
Employee Other Income is a document to declare other income of an employee from different sources.